src/control/Control.cpp

```diff
@@ -1,5 +1,9 @@
 #include "Control.h"
 
+#ifdef ENABLE_AUDIO
+#include "AudioControllerImpl.h"
+#endif
+
```

Comment:
Avoid to many preprozessor trees in different files by moving all into the AudioControler.cpp